src/tracing/backend.h

```diff
@@ -64,13 +64,13 @@ class Backend : public ::chip::IntrusiveListNodeBase<>
     virtual void TraceInstant(const char * label, const char * group) {}
 
     virtual void TraceCounter(const char * label) {}
-    virtual void TraceMetric(const char * label, int32_t value) {}
     virtual void LogMessageSend(MessageSendInfo &) { TraceInstant("MessageSent", "Messaging"); }
     virtual void LogMessageReceived(MessageReceivedInfo &) { TraceInstant("MessageReceived", "Messaging"); }
 
     virtual void LogNodeLookup(NodeLookupInfo &) { TraceInstant("Lookup", "DNSSD"); }
     virtual void LogNodeDiscovered(NodeDiscoveredInfo &) { TraceInstant("Node Discovered", "DNSSD"); }
     virtual void LogNodeDiscoveryFailed(NodeDiscoveryFailedInfo &) { TraceInstant("Discovery Failed", "DNSSD"); }
+    virtual void LogEvent(MetricEvent &) { TraceInstant("Metric Event", "Metric"); }
```

Comment:
what is a `MetricEvent` ? where is the corresponding include for this?

Reply:
I now see it was added the same as the `Info` above. This does seem like the name is very generic though. All the others seem to be a bit more specific (discovered, lookup etc) however `MetricEvent` does not seem as clear.

Reply:
What is the action here?

Reply:
I could use LogMetricEvent to be explicit. But given it is C++ method, overloading did not need to have name of the type in the method name.

Reply:
Please use `LogMetricEvent` since there is an existing widely used `LogEvent` that is a completely different context. It's not that much typing.

Reply:
Renamed to LogMetricEvent